styles.css

```diff
@@ -46,6 +46,8 @@ body{
     border-radius: 10px;
     font-size: 25px;
     margin-bottom: 40px;
+    margin-bottom: 40px;
+
   }
```

Comment:
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!. I noticed an unexpected empty line before the closing brace and I suggest it's best you delete it to applie [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).